src/server/scripts/Outland/TempestKeep/Mechanar/boss_mechano_lord_capacitus.cpp

```diff
@@ -134,9 +139,19 @@ class boss_mechano_lord_capacitus : public CreatureScript
                             break;
                         case EVENT_SUMMON_NETHER_CHARGE:
                         {
-                            Position pos = me->GetRandomNearPosition(5.0f);
-                            me->SummonCreature(NPC_NETHER_CHARGE, pos, TEMPSUMMON_TIMED_DESPAWN, 18000);
-                            events.ScheduleEvent(EVENT_SUMMON_NETHER_CHARGE, 10s);
+                            uint32 spellId;
+                            uint32 netherChargeTimer;
+                            netherChargeTimer = DUNGEON_MODE(urand(9000, 11000), urand(2000, 5000));
+                            switch (urand(0, 3))
+                            {
+                                case 0: spellId = SPELL_SUMMON_NETHER_CHARGE_NE; break;
+                                case 1: spellId = SPELL_SUMMON_NETHER_CHARGE_NW; break;
+                                case 2: spellId = SPELL_SUMMON_NETHER_CHARGE_SE; break;
+                                default:
```

Comment:
no need for default here. If all possible cases are between 0 and 3 you can just go with cases.